Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -85,39 +101,57 @@ namespace ROS2
 
     void ROS2SpawnerComponent::SpawnEntity(const SpawnEntityRequest request, SpawnEntityResponse response)
     {
-        AZStd::string_view key(request->name.c_str(), request->name.size());
+        AZStd::string_view spawnable_name(request->name.c_str(), request->name.size());
+        AZStd::string_view spawn_point_name(request->xml.c_str(), request->xml.size());
```

Comment:
A comment here about repurposing the field would be much needed. Also, we could actually use the xml in a more standard way and expect something like
<spawn_point_name>orchard_row_44</spawn_point_name>

Reply:
For now I did not use proper XML here not to complicate the code unnecessarily, but I added a todo since using this format might be useful in the future.